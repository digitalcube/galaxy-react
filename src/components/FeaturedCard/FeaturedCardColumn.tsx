import React, { FC, PropsWithChildren, useMemo } from 'react';

export type FeaturedCardColumnProps = PropsWithChildren<{
  column?: number;
}>;
export const FeaturedCardColumn: FC<FeaturedCardColumnProps> = ({
  children,
  column = 3,
}) => {
  const colWidth = useMemo(() => {
    const result = Math.round(12 / column);
    if (result > 12) return 12;
    if (result <= 1) return 1;
    return result;
  }, [column]);
  return (
    <div className="shifter-guides">
      <div className="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg">
        <div className="row">
          {Array.isArray(children) ? (
            <>
              {children.flat().map((child, index) => (
                <div className={`col-md-${colWidth}`} key={index}>
                  {child}
                </div>
              ))}
            </>
          ) : (
            <div className={`col-md-${colWidth}`}>{children}</div>
          )}
        </div>
      </div>
    </div>
  );
};
