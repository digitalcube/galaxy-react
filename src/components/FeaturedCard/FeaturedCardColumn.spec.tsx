import { render } from '@testing-library/react';
import React from 'react';
import { Default as FeaturedCardColumn } from './FeaturedCardColumn.stories';

describe('FeaturedCardColumn', () => {
  it.each([
    [0, 0],
    [1, 1],
    [0, 5],
    [12, 12],
    [13, 12],
    [100, 12],
  ])('to match snapshot [column %p | item %p]', (column, items) => {
    const { asFragment } = render(
      <FeaturedCardColumn column={column} items={items} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
