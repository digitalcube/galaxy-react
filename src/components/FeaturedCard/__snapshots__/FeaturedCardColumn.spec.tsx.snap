// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedCardColumn to match snapshot [column 0 | item 0] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`FeaturedCardColumn to match snapshot [column 0 | item 5] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`FeaturedCardColumn to match snapshot [column 1 | item 1] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-12"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`FeaturedCardColumn to match snapshot [column 12 | item 12] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`FeaturedCardColumn to match snapshot [column 13 | item 12] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`FeaturedCardColumn to match snapshot [column 100 | item 12] 1`] = `
<DocumentFragment>
  <div
    class="shifter-guides"
  >
    <div
      class="container-fluid mb-4 pb-4 px-0 guides-list guides-list-lg"
    >
      <div
        class="row"
      >
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
        <div
          class="col-md-1"
        >
          <div
            class="pt-4 px-4 h-100 position-relative guide-list-item"
          >
            <h3
              class="mt-0 font-weight-bold guide-list-title"
            >
              Read the docs
            </h3>
            <div
              class="position-absolute guide-list-anchor"
            >
              <a
                class="px-3 py-0 rounded btn-bordered btn-smaller d-inline-flex align-items-center font-weight-bold btn btn-outline-primary"
                href="https://getshifter.io"
                rel="noopener"
                target="_blank"
              >
                Explore Shifter docs
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
